src/Honeybee_Add to EnergyPlus Library.py

```diff
@@ -50,7 +50,32 @@
 
 import scriptcontext as sc
 import Grasshopper.Kernel as gh
+import os
 
+def prepEPObjects(rawStrings):
```

Comment:
Hi @MingboPeng! This method can fail if the EnergyPlus file is not clean. Instead use Honeybee's `importEPLibraryFromFile` method.